packages/article/src/Infrastructure/Symfony/HttpKernel/SuluArticleBundle.php

```diff
@@ -275,7 +275,7 @@ public function prependExtension(ContainerConfigurator $container, ContainerBuil
                         ],
                     ],
                     'templates' => [
-                        'articles' => [
+                        ArticleInterface::TEMPLATE_TYPE => [
```

Comment:
I personally would remove the "template_type" currently singleton (page, snippet, article) and replace it with the resource key (pages, snippets, articles) but that should be discussed and tackled in an own Pull request.